libraries/lib-wave-track/Sequence.cpp

```diff
@@ -858,7 +848,7 @@ bool Sequence::HandleXMLTag(const std::string_view& tag, const AttributesList &a
       }
 
       mBlock.push_back(wb);
-
+      mBlockCount.store(mBlock.size(), std::memory_order_release);
```

Comment:
Is it necessary? I guess that should be enough to update `mBlockCount` in `HandleXMLEndTag`?

Reply:
Yes, I definitely overdid it here.